data/tlg0031/tlg007/tlg0031.tlg007.perseus-grc2.xml

```diff
@@ -1,14 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<?xml-model href="http://www.stoa.org/epidoc/schema/8.19/tei-epidoc.rng"
-schematypens="http://relaxng.org/ns/structure/1.0"?>
-<?xml-model href="http://www.stoa.org/epidoc/schema/8.19/tei-epidoc.rng"
-schematypens="http://purl.oclc.org/dsdl/schematron"?>
-<TEI xmlns="http://www.tei-c.org/ns/1.0">
-    <teiHeader>
+<?xml-model href="https://epidoc.stoa.org/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?> 
+<TEI xmlns="http://www.tei-c.org/ns/1.0"> 
+    <teiHeader xml:lang="eng">
         <fileDesc>
             <titleStmt>
-                <title>New Testament - 1 Corinthians</title>
-                
+                <title xml:lang="grc">Κορίνθιους Α'</title>                
```

Comment:
I can't tell but this does not look like ′ but rather a generic straight single quote? should be Α′